README.md

```diff
@@ -603,6 +546,71 @@ ddosify -config ddosify_config_dynamic.json
     ]
 }
 ```
+
+## Assertion
+
+Ddosify supports assertion on `status code`, `response body`, `response size`, `response time`, `headers` and `variables`. You can use `assertion` parameter on config file to check if the response matches the given condition. If the condition is not met, Ddosify will fail the scenario.
```

Comment:
At default, Ddosify marks the step result as successful if it sends the request and receives the response without any network error happening. Status code or body type (or content) does not have any effect on success/failure criteria. But this may not be a good test result for your use case and you may want to create your success/fail logic.  That's where you can use Assertions.

Ddosify supports assertion on `status code`, `response body`, `response size`, `response time`, `headers`, and `variables`. You can use the `assertion` parameter on the config file to check if the response matches the given condition per step. If the condition is not met, Ddosify will fail the step. Check the [example config]([config_examples/config.json](https://github.com/ddosify/ddosify/blob/master/config_examples/config.json)) to see how it looks.

As shown in the related table the first 5 keywords store different data related to the response. The last keyword `variables` stores the current state of environment variables for the Step. You can use `Functions` or `Operators` to build conditional expressions based on these keywords.

You can write multiple assertions for a step. If any assertion fails, the step is marked as Failed.

If Ddosify can't receive the response for a request, that step is marked as Failed without processing the assertions. You will see a **Server Error** as a failure reason on the test result instead of an **Assertion Error**.